Source/mainmenu.cpp

```diff
@@ -6,6 +6,7 @@
 #include "all.h"
 #include "../3rdParty/Storm/Source/storm.h"
 #include "../DiabloUI/diabloui.h"
+#include "misc\config.h" //For re-reading options from config

```

Comment:
There is at least one of this left.

Reply:
![image](https://images-na.ssl-images-amazon.com/images/I/819TInppvbL._AC_SX522_.jpg)

Reply:
Fixed. I did a search, and I couldn't find any other instances in the code.